utils/vlog/tests/muxes/golden.pb_type.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Can we add a comment that lists the source file that generated this file?  I'm having trouble linking where this XML was generated from.

Maybe:
``<-- Generate from mux_gen.py <args list> -->``

Reply:
@kgugala

Reply:
this was actually printed in xmls. I've added more info to be printed there (all the script arguments)

However, the golden pb_type files does not have those annotations - in most cases they are not generated.